src/dune_rules/pkg_toolchain.ml

```diff
@@ -65,10 +65,28 @@ let ocaml context env ~bin_dir =
   Ocaml_toolchain.make context ~which ~env ~get_ocaml_tool
 ;;
 
-(* The path to the directory containing the artifacts within the
-   temporary install directory. When installing with the DESTDIR
-   variable, the absolute path to the final installation directory is
-   concatenated to the value of DESTDIR. *)
+(* Returns the path to the directory containing the artifacts within the
+   temporary install directory. When installing with the DESTDIR variable, the
+   absolute path to the final installation directory is concatenated to the
+   value of DESTDIR. That is, the artifacts will be at a path like
+   "/tmp/dune-toolchain-destdir_RRR_ocaml-base-compiler.5.2.1-XXXXXXXX/home/user/.cache/dune/toolchains/ocaml-base-compiler.5.2.1-XXXXXXXX/target"
```

Comment:
```suggestion
   "/tmp/dune-toolchain-destdir_RRR_ocaml-base-compiler.5.2.1-XXXXXXXX/$XDG_CACHE_DIR/dune/toolchains/ocaml-base-compiler.5.2.1-XXXXXXXX/target"
```

Reply:
For the paths in this comment I deliberately left them as "/home/user" to clarify that it's just a string. The goal is that if you're debugging this code to know what sort of path to expect. In this case in particular I think it's important to use a literal example path without the variable because it's unusual to have "/home/user" in a path but not at the root of the path, so I want to clarify that that is indeed what is happening here. Also in some environments such as macos and windows, `XDG_CACHE_DIR` isn't set, and dune tries to guess the appropriate path, and I expect readers to be more likely to recognize the "/home/user" convention than to know about XDG variables.